mysite/templates/SilverStripe/CommentNotifications/BlogCommentEmail.ss

```diff
@@ -0,0 +1,403 @@
+<!-- Inliner Build Version 4380b7741bb759d6cb997545f3add21ad48f010b -->
```

Comment:
What does this file add? Is it something that should be in the blog or comments module instead?

Reply:
Direct carry over from `cwp/installer` which configured the blog & comments modules out of the box. If it's unneeded then we can omit it I think.
There is a default `CommentEmail` template in `silverstripe/comments`, to be fair I kinda blindly made the assumption that this was required at some level. I've not inspected it closely to see what it does differently, if anything.

Reply:
Interesting. It's probably worth testing it against the default to see where it should be moved to, if not where it here.

Reply:
I'd rather omit this for now and reintroduce it later if it's necessary to make a SilverStripe (not CWP) blog recipe function

Reply:
We built this because CWP needed a specific look and feel. I guess we could leave it out for now.

Reply:
I think that's agreeable since this is a `silverstripe/` recipe.